config/locales/en.yml

```diff
@@ -3159,6 +3159,15 @@ en:
             answer_html: |
               An email address, ID, proof of income and/or benefits, and your child's Social Security Number.<br/><br/>
               Children claimed for the Child Tax Credit must have a valid Social security number (SSN), however the tax filer (a parent, grandparent, or guardian) can have either a SSN or an Individual Taxpayer Identification Number (ITIN).
+          check_payment_status:
+            question: "Where can I check the status of my payments?"
```

Comment:
I know there was some conversation about not including the html styling in the YML but I figured this was okay because the rest of answer_htmls include html, it would require me to change the layout and the spanish was already included